app/viewmodels/models/PageViewModel.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.models
+
+import play.api.mvc.Call
+import viewmodels.DisplayMessage
+import viewmodels.DisplayMessage.{InlineMessage, Message}
+
+case class PageViewModel[+A](
+  title: Message,
+  heading: InlineMessage,
+  description: Option[DisplayMessage],
+  inset: Option[DisplayMessage],
```

Comment:
should description and inset be part of the underlying page model instead as they are optional? A case could be made for description as most pages have it but not sure about inset